ConfigurationSystem/Agent/VOMS2CSAgent.py

```diff
@@ -0,0 +1,323 @@
+"""
+  VOMS2CSAgent performs the following operations:
+
+    - Adds new users for the given VO taking into account the VO VOMS information
+    - Updates the data in the CS for existing users including DIRAC group membership
+    -
+"""
+
+__RCSID__ = "$Id$"
+
+import os
+from DIRAC.Core.Base.AgentModule                       import AgentModule
+from DIRAC.ConfigurationSystem.Client.CSAPI            import CSAPI
+from DIRAC.FrameworkSystem.Client.NotificationClient   import NotificationClient
+from DIRAC.Core.Security.VOMSService                   import VOMSService
+from DIRAC                                             import S_OK, S_ERROR, gConfig
+from DIRAC.ConfigurationSystem.Client.Helpers.Registry import getVOOption, getUserOption, \
+                                                              getVOMSRoleGroupMapping, getUsersInVO, \
+                                                              getAllUsers
+from DIRAC.Core.Utilities.Proxy                        import executeWithUserProxy
+
+class VOMS2CSAgent( AgentModule ):
+
+  def initialize( self ):
+    self.am_setOption( "PollingTime", 3600 * 6 ) # Every 6 hours
```

Comment:
Why here and not in ConfigTemplate.cfg? If it's here is hidden to the administrator.